.github/workflows/asset_compilation.yml

```diff
@@ -0,0 +1,135 @@
+        REPORTING_DATABASE_DB_TEST: reporting_test
+      run: |
+        apk add --no-cache \
+          nodejs yarn npm \
+          tzdata \
+          git \
+          bash \
+          freetds-dev \
+          icu icu-dev \
+          curl libcurl curl-dev \
+          imagemagick \
+          libmagic file-dev file \
+          build-base libxml2-dev libxslt-dev postgresql-dev \
+          libgcc libstdc++ libx11 glib libxrender libxext libintl ttf-dejavu ttf-droid ttf-freefont ttf-liberation ttf-freefont \
+          chromium nss freetype freetype-dev harfbuzz ca-certificates ttf-freefont \
+          lftp postgresql tmux postgis geos geos-dev \
+          shared-mime-info \
+          python3 py3-pip
+
+        pip3 install awscli
+
+        echo "postgres:5432:*:postgres:postgres" > ~/.pgpass
+        chmod 600 ~/.pgpass
+
+        gem install bundler --version=2.3.8
+
+    # According to https://www.jessesquires.com/blog/2021/08/23/caching-bundler-on-github-actions/
+    # this is fragile and failure prone, but the step they recommend using instead (ruby/ruby-setup)
+    # isn't supported on Alpine.
+    - name: cache gems
+      uses: actions/cache@v3
+      id: gemcache
+      with:
+        path: |
+          vendor/bundle
+          /usr/local/bundle/
+        key: ${{ runner.os }}-gemcache-${{ hashFiles('**/Gemfile.lock') }}-${{ hashFiles('**/.ruby-version') }}
+        # restore-keys: |
+        #   ${{ runner.os }}-gem-bundler-
+    # - name: Ruby
+    #   uses: ruby/setup-ruby@v1
+    #   with:
+    #     bundler-cache: true
+
+    - name: Install gems
+      if: always()
+      run: |
+        bundle config set --local without 'production staging development'
+        bundle install --jobs 10 --retry 3
+
+    - name: Generate checksum and compile
+      env:
+        SECRETS_YML_SECRET_ARN: ${{ secrets.SECRETS_YML_SECRET_ARN }}
+        DATABASE_ADAPTER: postgresql
+        DATABASE_HOST: postgres
+        DATABASE_PASS: postgres
+        DATABASE_USER: postgres
+        DEFAULT_FROM: greenriver.testing@mailinator.com
+        ENCRYPTION_KEY: strongEncryptionstrongEncryptionstrongEncryption
+        HEALTH_DATABASE_ADAPTER: postgresql
+        HEALTH_DATABASE_HOST: postgres
+        HEALTH_DATABASE_PASS: postgres
+        HEALTH_DATABASE_USER: postgres
+        HEALTH_FROM: greenriver.testing@mailinator.com
+        REPORTING_DATABASE_ADAPTER: postgresql
+        REPORTING_DATABASE_HOST: postgres
+        REPORTING_DATABASE_PASS: postgres
+        REPORTING_DATABASE_USER: postgres
+        WAREHOUSE_DATABASE_ADAPTER: postgis
+        WAREHOUSE_DATABASE_HOST: postgres
+        WAREHOUSE_DATABASE_PASS: postgres
+        WAREHOUSE_DATABASE_USER: postgres
+        WAREHOUSE_DATABASE_DB: warehouse_db
+        HEALTH_DATABASE_DB: health_db
+        REPORTING_DATABASE_DB: reporting_db
+        DATABASE_DB_TEST: boston_cas_test
+        DATABASE_WAREHOUSE_DB_TEST: test_hmis_warehouse
+        HOSTNAME: openpath.host
+        PORT: 80
```

Comment:
@eanders Was hoping it would be easy for you to eyeball which of these we actually need and delete the rest?

Reply:
I don't think any of those are in use for asset recompilation but I'm unsure if you could run `bundle exec bin/compile_assets.rb` without them.  Maybe try getting rid of everything but `HOSTNAME`?

Reply:
Oh, sorry, just which ones are relevant to CAS.

Reply:
Yup, thank you 🙏